docs/content/guides/major-pg-upgrade.md

```diff
@@ -0,0 +1,242 @@
+[backup]({{< relref "tutorial/backup-management.md" >}}). This will offer another layer of data 
+protection in cases where the upgrade process does not complete as expected.
+
+### Step 2: Scale Down Replicas
+
+PGO needs to identify the primary database instance before the running your upgrade because the primary
+pgdata volume will need to be identified and mounted to the upgrade Job. Any cluster replicas will not be
+upgraded and will need to be recreated after the upgrade and post-upgrade tasks are completed. To ensure
+errors are avoided, any replicas must be scaled down before initiating the upgrade process. 
+
+Scaling down is simple, your manifest should be configured to have only one instance named under `spec.instances`
+and the `replicas` value, if set, should be equal to `1`. For example, if your existing configuration is
+
+```
+spec:
+  instances:
+    - name: instancea
+      replicas: 3
+      dataVolumeClaimSpec:
+        accessModes:
+        - "ReadWriteMany"
+        resources:
+          requests:
+            storage: 1Gi
+    - name: instanceb
+      replicas: 2
+      dataVolumeClaimSpec:
+        accessModes:
+        - "ReadWriteMany"
+        resources:
+          requests:
+            storage: 1Gi
+```
+
+You will need to set it to something similar to 
+
+```
+spec:
+  instances:
+    - name: instancea
+      replicas: 1
+      dataVolumeClaimSpec:
+        accessModes:
+        - "ReadWriteMany"
+        resources:
+          requests:
+            storage: 1Gi
+```
+
+then apply the changes with
+
+```
+kubectl apply -k examples/postgrescluster
+```
+
+and wait for them to complete before continuing on to the next steps.
+
+<!-- TODO(tjmoore4): This step should not be required after follow on work to run pgBackRest stanza 
+upgrade and backup automatically during upgrade process. This will allow the replicas to be automatically
+scaled down before the upgrade and back up once the post-upgrade steps are completed. -->
+
+### Step 3: Configure the PostgresCluster
+
+At this point, your running cluster is ready for the major upgrade. As shown below, you will need to 
+configure the `spec.upgrade` section to enable the major upgrade. This section will also define the current 
+(i.e. pre-upgrade) Postgres version of the cluster and define the `crunchy-upgrade` image, assuming the
+relevant environment variable, `RELATED_IMAGE_PGUPGRADE`, is not being utilized. 
+
+At the same time, you will need to set the `spec.postgresVersion` and `spec.image` values to reflect the
+version the PostgresCluster will be after the upgrade is completed. Below, you'll see an example configuration
+for a {{< param fromPostgresVersion >}} to {{< param postgresVersion >}} upgrade:
+
+```
+spec:
+  upgrade:
+    enabled: true
+    fromPostgresVersion: {{< param fromPostgresVersion >}}
+    image: {{< param imageCrunchyPGUpgrade >}}
+  image: {{< param imageCrunchyPostgres >}}
+  postgresVersion: {{< param postgresVersion >}}
+```
+
```

Comment:
What do you think about breaking this up into two example sections? First you have lines 84-7 describing the spec.upgrade field, then give the spec.upgrade example. Then you have (the current) lines 89-91 and then only have the spec.image and spec.postgresVersion fields.

I like examples that are clearly explained by the nearest text, BUT I also think we can make a mistake by not showing the whole manifest, so I'm on the fence here.